webconsole/src/main/java/org/secondbase/webconsole/widget/Widget.java

```diff
@@ -0,0 +1,11 @@
+package org.secondbase.webconsole.widget;
+
+import com.sun.net.httpserver.HttpHandler;
+
+/**
+ * Implemented by WebConsole widgets.
```

Comment:
Maybe say "Interface which webconsole widgets have to implement" or something along these lines?